Lib/test/test_time.py

```diff
@@ -441,7 +441,7 @@ def test_mktime(self):
                          "disabled because of a bug in glibc. Issue #13309")
     def test_mktime_error(self):
         # It may not be possible to reliably make mktime return error
-        # on all platfom.  This will make sure that no other exception
+        # on all platform.  This will make sure that no other exception
```

Comment:
I think this should be "make mktime return an error on all platforms": "error" -> "an error", and "platform" -> "platforms".